### ISSUE TITLE
Transmission client issues

### ISSUE THREAD

**Issue by Tronicgg** [ManiMatter/decluttarr#57]
Hi! Having issues with Transmission client. Running docked. Basically no torrent gets detected as stalled/missing metadata so no torrent gets removed. I've been going over this for a couple of days but can't find the cause. You mention on documentation that it does work with other torrents minus specific qBitorrent features. Or maybe I misread.

Log on verbose mode just spits out endless loop:

decluttarr  | [VERBOSE]: Cleaning queue on Radarr:
decluttarr  | [VERBOSE]: >>> Queue is clean.

Radarr and Sonarr do get an ok:

decluttarr  | [INFO   ]: OK | Radarr
decluttarr  | [INFO   ]: OK | Sonarr

Wanted to narrow down if my install or something was being messed up so installed qBitorrent and sure enough it does work perfectly with it. Anything special I have to do to make it run on Transmission?

**Comment by ManiMatter** [comment]
Thank you for raising this. Even if you use trasmission the removal should work; tag-protection is not working but should not matter here

could you please
1) switch to debug mode
2) turn off all remove functions but one where you expect a removal
3) let it run until the supposed remove should be trigged
4) paste the full logs to a pastebin
5) paste a screenshot of qbit & sonarr to a pasteimg

will try to have a look on the weekend

**Comment by Tronicgg** [comment]
Had to switch all apps to qBitorrent. I'll go back to Transmission during the weekend and paste logs here. Thanks!

**Comment by ManiMatter** [comment]
I just added more logs to :dev image - suggest you use that one when pasting the logs

**Comment by ManiMatter** [comment]
hi, do you still have issues or can this be closed?

**Comment by ManiMatter** [comment]
Closing due to inactivity

**Comment by cliny** [comment]
I have switched back to transmission and I am having this exact issue. If transmission grabs the metadata and it stalls, decluttarr will remove it (based on "remove slow" I believe). However, if it doesn't grab the metadata, "remove metadata" doesn't remove it, possibly because the state is also "downloading".

See attached debug log, I have included all removal options, and left 1 torrent from lidarr (Magnetized transfer - retrieving metadata 0%)

https://pastebin.com/jFfxdwGA

Thanks
